app/components/ui/Modal/ServiceDiscoveryModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { BaseModal } from './Modal';
+import * as dataService from '../../../utils/DataService';
+
+import styles from './ServiceDiscoveryModal.scss';
+
+
+class ServiceDiscoveryModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      eligibilities: [],
+      subcategories: [],
+      checkedItems: {},
+    };
+  }
+
+  componentDidMount() {
+    const { categoryId } = this.props;
+
+    dataService.get(`api/eligibilities?category_id=${categoryId}`).then(response => {
+      const { eligibilities } = response;
+      this.setState({
+        eligibilities,
+      });
+    });
+
+    dataService.get(`api/categories/subcategories?id=${categoryId}`).then(response => {
+      const { categories } = response;
+      this.setState({
+        subcategories: categories,
+      });
+    });
```

Comment:
I originally implemented this so it just fetched whichever one of these is relevant to the modal (ie if we are showing eligibilities or subcategories). But then I realized that if we are using this same modal for david's view, we'll need to fetch both anyway.